pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -63,13 +63,12 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 					DescriptionAnnotation: SecurityContextConstraintPrivilegedDesc,
 				},
 			},
-			AllowPrivilegedContainer:  true,
-			AllowHostDirVolumePlugin:  true,
-			AllowEmptyDirVolumePlugin: true,
-			AllowHostNetwork:          true,
-			AllowHostPorts:            true,
-			AllowHostPID:              true,
-			AllowHostIPC:              true,
+			AllowPrivilegedContainer: true,
+			Volumes:                  []kapi.FSType{kapi.FSTypeHostPath, kapi.FSTypeEmptyDir, kapi.FSTypeSecret},
```

Comment:
No, this was just to get all the mechanics worked out by replacing the existing bools.  We can open this list up